pypeerassets/provider/holytransaction.py

```diff
@@ -55,7 +55,7 @@ def network_long_to_short(name):
     def network_short_to_long(name):
         """convert short network name like "ppc" to "peercoin". """
 
-        if len(name) > 3:
+        if len(name) < 4:
```

Comment:
<= 4
Example,
'tppc' will result to length 4 which would not be considered if conditional is only < .